stable/dask-distributed/README.md

```diff
@@ -0,0 +1,87 @@
+# Dask Distributed Helm Chart
+
+Dask Distributed allows distributed computation in Python the chart also includes a single user Jupyter Notebook
+
+* https://github.com/dask/distributed
+* http://jupyter.org/
+
+## Chart Details
+This chart will do the following:
+
+* 1 x Dask Scheduler with port 8786 (scheduler) and 8786 (webUI) exposed on an external LoadBalancer
```

Comment:
The web UI may be exposed on port 8787 rather than 8786 as explained here.  Alternatively, we might also consider hosting on normal port 80.  We usually choose 8787 because normal users don't want to run with the privileges necessary to host on ports < 1024.  However in a Kubernetes setting this may not be as big a deal?